### PR TITLE
Print currently livesynced file name

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
When using `appbuilder livesync platform --watch` the name of the file that is synced is not displayed.

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:broken_heart:

**Comment by justcodebuilduser** [comment]
:heart:

File: services/livesync-service-base.ts [hunk 1]
```diff
@@ -245,10 +246,17 @@ class LiveSyncServiceBase implements ILiveSyncServiceBase {
 			} else {
 				deviceAppData.device.fileSystem.transferFiles(deviceAppData, localToDevicePaths).wait();
 			}
-			this.$logger.info("Successfully transferred all project files.");
+
+			this.logFilesSyncInformation(localToDevicePaths, (file: Mobile.ILocalToDevicePathData) => { this.$logger.info(`Successfully transferred ${path.basename(file.getLocalPath()).yellow}`); });
 		}).future<void>()();
 	}
 
+	private logFilesSyncInformation(localToDevicePaths: Mobile.ILocalToDevicePathData[], action: (file: Mobile.ILocalToDevicePathData) => void): void {
```
Comment (rosen-vladimirov) [inline services/livesync-service-base.ts]: as both times when this method is used, you are passing $logger.info, I would recommend to change the second parameter to string and use `util.format`:
```TypeScript
private logFilesSyncInformation(localToDevicePaths: Mobile.ILocalToDevicePathData[], messagePerFile: string): void {
	_.each(localToDevicePaths, (file: Mobile.ILocalToDevicePathData) => {
		this.$logger.info(util.format(message, path.basename(file.getLocalPath())).yellow);
	});
}
```
and call the method with:
```TypeScript
this.logFilesSyncInformation(localToDevicePaths, "Transferring %s.");
this.logFilesSyncInformation(localToDevicePaths, "Successfully transferred %s.");
```

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**